.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+
+python:
+ - "3.7-dev"
+ - "3.6"
+ - "3.5"
+ - "3.4"
+ - "3.3"
+ - "3.2"
+ - "2.7"
+ - "2.6"
+
+script:
+ # Simple test runs
+ - ./local-install.sh
+ - ./curl-install.sh
+
+after_script:
+ # Static analysis: for info, won't fail the build
```

Comment:
Well, maybe it should fail the build.

Reply:
there's just a few linting errors, so why don't we get rid of them and make sure they won't happen to be repeated?